lab-trent/routes/statshotroute.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const express = require('express');
+const createError = require('http-errors');
+
+const StatSnapshot = require('../model/statsnapshot');
+const resourceManager = require('../lib/resourcemanager');
+
+const router = new express.Router();

```

Comment:
You don't need to instantiate the Router object with `new`.

Also, it's okay with a singular resource and associated routes, but if you add another modular routes file, you'll want to utilize the same Router object in both, so you'd want to move the Router to the server, and inject it into each router module using dep injection.